Vehicle/VehicleModel/doc/spec.md

```diff
@@ -0,0 +1,122 @@
+        + (`agriculturalVehicle`, `bicycle`, `bus`, `car`, `caravan`,
+           `carWithCaravan`, `carWithTrailer`, `lorry`, `moped`,
+           `motorcycle`, `motorcycleWithSideCar`, `motorscooter`, `tanker`, `trailer`, `van`)
+    + Mandatory
+
++ `brandName` : Vehicle's brand name.
+    + Attribute type: [Text](https://schema.org/Text)
+    + See also: [https://schema.org/brand](https://schema.org/brand)
+    + Mandatory
+
++ `modelName` : Vehicle's model name.
+    + Attribute type: [Text](https://schema.org/Text)
+    + See also: [https://schema.org/model](https://schema.org/model)
+    + Mandatory
+
++ `manufacturerName` : Vehicle's manufacturer name.
+    + Attribute type: [Text](https://schema.org/Text)
+    + See also: [https://schema.org/model](https://schema.org/model)
+    + Mandatory
+
++ `vehicleModelDate` : The release date of a vehicle model (often used to differentiate versions of the same make and model).
+    + Normative References: [https://schema.org/vehicleModelDate](https://schema.org/vehicleModelDate)
+    + Optional
+
++ `cargoVolume` : The available volume for cargo or luggage. For automobiles, this is usually the trunk volume.
+    + Normative References: [https://schema.org/cargoVolume](https://schema.org/cargoVolume)
+    + Default Unit: Liters
+    + Optional
+    + Note: If only a single value is provided (type Number) it will refer to the maximum volume.
+    
++ `fuelType` : The type of fuel suitable for the engine or engines of the vehicle.
+    + Normative References: [https://schema.org/fuelType](https://schema.org/fuelType)
+    + Allowed values: one Of (`gasoline`, `petrol(unleaded)`, `petrol(leaded)`, `petrol`, `diesel`, `electric`,
+    `hydrogen`, `lpg`, `autogas`, `cng`, `biodiesel` `ethanol`, `hybrid electric/petrol`, `hybrid electric/diesel`, `other`)
+    + Optional
+
++ `fuelConsumption` : The amount of fuel consumed for traveling a particular distance or temporal
+duration with the given vehicle (e.g. liters per 100 km).
+    + Normative References: [https://schema.org/fuelConsumption](https://schema.org/fuelConsumption)
+    + Default unit: liters per 100 kilometer. 
+    + Optional
+
++ `height` : Vehicle's height.
+    + Normative References: [https://schema.org/height](https://schema.org/height)
+    + Optional
+
++ `width` : Vehicle's width.
+    + Normative References: [https://schema.org/width](https://schema.org/width)
+    + Optional
+
++ `depth` : Vehicle's depth.
+    + Normative References: [https://schema.org/width](https://schema.org/depth)
+    + Optional
+
++ `weight` : Vehicle's weight.
+    + Normative References: [https://schema.org/weight](https://schema.org/weight)
+    + Optional
+
++ `vehicleEngine` : Information about the engine or engines of the vehicle.
+    + Normative References: [https://schema.org/vehicleEngine](https://schema.org/vehicleEngine)
+    + Optional
+    + Note: This property could be at vehicle level as well.
```

Comment:
well there can be a tuned vehicle ... which overwrites the default engine's model ;)